### PR TITLE
Fix median depth calculation and add to report

### PR DESCRIPTION
**Pull request by safay**
Using a sample that exposed this issue for testing, a negative control with high read depth (1.6M reads): https://app.onecodex.com/classification/4c2ae2e6b5cc40af

A diff of the `.report.json.gz` shows that this fixes the median depth reporting (and changes the mean depth exposed in the report):

image

This also fixes a previously unknown issue, a warning about zero coverage bases that was not getting raised for samples like these:

image

Added `median_depth` to the pdf report:
image

vs. original report:
image

### CODE REVIEW

File: report.ipynb [hunk 1]
```diff
@@ -966,7 +966,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.1"
+   "version": "3.10.0"
```
Comment (safay) [inline report.ipynb:969]: Not sure why this and the change in line 955 happened.

File: README.md [hunk 2]
```diff
@@ -41,14 +41,12 @@ docker \
...
   --env reference=reference/nCoV-2019.reference.fasta \
-  --env input_fastq=data/twist-target-capture/RNA_control_spike_in_10_6_100k_reads.fastq.gz \
   --env primer_bed_file=reference/artic-v1/ARTIC-V3.bed \
   covid19 \
-  jobscript.sh
+  jobscript.sh \
+  data/twist-target-capture/RNA_control_spike_in_10_6_100k_reads.fastq.gz
 ```
 
-This currently produces a `consensus.fa` file, a `variants.vcf`, a BAM file (`covid19.bam`), nextstrain results (`nextstrain.json`) and pangolin results (`pangolin.csv`).
```
Comment (audy) [inline README.md:50]: was this change intentional?

**Review by audy — APPROVED** [review]
